discord/interactions.py

```diff
@@ -744,6 +799,8 @@ async def send_message(
             The file to upload.
         files: List[:class:`File`]
             A list of files to upload. Must be a maximum of 10.
+        return_message: :class:`bool`
+            Indicates if the message object should be returned instead of the interaction object.
```

Comment:
Possibly note that this uses a second API request

Reply:
> > Possibly note that this uses a second API request
> 
> Shouldn't that be noted for passing `view` too?

Yes, but ideally there shouldn't be an API call made for that at all since the message (or at least part of it) is received in the interaction payload